packages/next/src/generators/application/schema.d.ts

```diff
@@ -11,10 +11,11 @@ export interface Schema {
   unitTestRunner?: 'jest' | 'none';
   e2eTestRunner?: 'cypress' | 'none';
   linter?: Linter;
-  skipWorkspaceJson?: boolean;
   js?: boolean;
   setParserOptionsProject?: boolean;
   standaloneConfig?: boolean;
   swc?: boolean;
   customServer?: boolean;
+  skipPackageJson?: boolean;
+  skipTsConfig?: boolean;
```

Comment:
Just like Angular, I don't think this should be optional. We probably need to have `tsconfig.base.json`

Reply:
done. remove flags skipTsConfig and skipPackageJson.